certbot/error_handler.py

```diff
@@ -19,6 +19,14 @@
              signal.SIGXCPU, signal.SIGXFSZ])
 
 
+class SignalExit(Exception):
```

Comment:
This class should be added to `errors.py` instead of this file since it can propagate out to the user.